lib/src/main/java/com/futurewei/alcor/common/enumClass/MessageType.java

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2019 The Alcor Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
@xieus - maybe we should start using the new license for new files? What do you think?

Reply:
It is okay. We will do a one-time change across the whole repo. Thanks for the reminder.